### DISCUSSION TITLE
GPU CONVERSION

### DISCUSSION THREAD

**Discussion by Underlevel** [Anjok07/ultimatevocalremovergui#589] [Q&A]
I can't check GPU conversion on my UVR5, can anyone tell what specs do I exactly need in order for me to check it? CPU conversion is just too slow.... Any help would be appreciated!

**Comment by kaswardy** [comment]
I also want to use my RTX 3070 GPU since on 10700k CPU, the rendering is very slow.

**Comment by kaswardy** [comment]
sorry... just saw there is a checkbox for GPU version right next to checkbox for instrument, vocals etc.

duh...
